src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -16,7 +16,7 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-@Describable("hudson.plugins.sshslaves.SSHLauncher")
+@Describable({"Launch agents via SSH", "hudson.plugins.sshslaves.SSHLauncher"})
```

Comment:
this is likely going to be an issue as there are multiple ways to "launch agents via SSH" (one is cloudbees specific with non blocking, and the other is the OSS ssh-slaves plugin.  we should always be able to use the class name of the describable to not be dependant on any naming changes.  has the UI changed such that the describable is no longer picked up correctly (should be ok as without it databinding will likely not work)

Reply:
It's changed to use https://github.com/jenkinsci/jenkins/pull/6464

Which is keyed by the loop index, https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/dropdownDescriptorSelector.jelly#L61